docs/writing-docs/doc-block-canvas.md

```diff
@@ -0,0 +1,73 @@
+---
+title: 'Canvas'
+---
+
+<div class="aside">
+
+💡 Currently, there's an issue when using MDX stories with IE11. This issue does <strong>not</strong> apply to [DocsPage](./docs-page.md). If you're interested in helping us fix this issue, read our
```

Comment:
Can we put this at the top of the MDX page and not on each of the DocBlocks page? I don't think it's a common enough problem to merit all the space. WDYT?

Reply:
Also I don't think it's something that can be fixed with a contribution since AFAIK it's a problem with the MDX library itself, not our implementation

Reply:
@shilman with the latest commit, I've removed the aside from all the docs blocks pages. I'll make a follow-up pull request polishing the wording on the MDX page. Sounds good?